API_Database.py

```diff
@@ -46,8 +49,9 @@ def Select_All():
     conn.commit()
     c.execute("SELECT * FROM API_Results ")
     Table = c.fetchall()
-    for i in Table:
-        print("Date: ", i[0])
-        print("Movie Title: ", i[1])
-        print("NASA URL: ", i[2])
-        print("WIKI URL: ", i[3])
\ No newline at end of file
+    return Table
```

Comment:
Could this be more structured than a list of tuples? In the template you are accessing by index which is hard to understand and can be error prone. https://docs.python.org/3/library/sqlite3.html#sqlite3.Row